src/CanService.cpp

```diff
@@ -11,12 +11,6 @@
 namespace VLCB
 {
 
-enum CanOpCodes
```

Comment:
I added these OpCodes at an early stage when I thought it would be good to group the OP-codes that belonged to each service. There wasn't much benefit to this idea so cleaning this up now.